### ISSUE TITLE
Warning in lighteval evaluation:Value is 0

### ISSUE THREAD

**Issue by Leawnn** [huggingface/open-r1#368]
hi,thank you for your projects!
When I evalutate deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B using the provided code on math-500, I cannot get the [Pred] ,The script I'm using is provided below:

```
MODEL=deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B
MODEL_ARGS="pretrained=$MODEL,dtype=bfloat16,max_model_length=32768,gpu_memory_utilisation=0.8"
OUTPUT_DIR=data/evals/$MODEL
# MATH-500
TASK=math_500
lighteval vllm $MODEL_ARGS "custom|$TASK|0|0" \
    --custom-tasks src/open_r1/evaluate.py \
    --use-chat-template \
    --output-dir $OUTPUT_DIR

```
I get WARNING but not an ERROR,such as :

![Image](https://github.com/user-attachments/assets/c9983c60-d854-4206-b797-95d5fae1e18c)
and the final result is 0,Anything wrong?
Look forward to you reply. Thanks!

**Comment by ZhangHandi** [comment]
same problem，is there any solution？

**Comment by stat-eklee** [comment]
same problem....

**Comment by huaian2004** [comment]
same issue after updated to latest

**Comment by KennyShang** [comment]
same problem....

**Comment by KennyShang** [comment]
Bug in  lighteval/src/lighteval/models/vllm/vllm_model.py 
When init the self.sampling_params there is a default max_tokens=16, so the max_new_tokens  never work. The output length fix to 16 token.

```
def _generate(
        self,
        inputs: list[list[int]],
        max_new_tokens: Optional[int] = None,
        stop_tokens: Optional[list[str]] = None,
        returns_logits: Optional[bool] = False,
        num_samples: int = 1,
        generate: bool = True,
    ) -> list[GenerativeResponse]:
        """Contains the actual logic of the generation."""
        sampling_params = self.sampling_params.clone() or SamplingParams()
        if generate:
            sampling_params.n = num_samples
            sampling_params.max_tokens = (
                max_new_tokens if sampling_params.max_tokens is None else sampling_params.max_tokens
            )
            sampling_params.stop = stop_tokens
            sampling_params.logprobs = 1 if returns_logits else 0
```

**Comment by huaian2004** [comment]
Thanks @KennyShang for the tips. With latest commit #392 , and run `pip install -e ".[dev]"` to reinstall all patches. Issue fixed.

I can get the result 0.838 with DeepSeek-R1-Distill-Qwen-1.5B
|      Task       |Version|     Metric     |Value|   |Stderr|
|-----------------|------:|----------------|----:|---|-----:|
|all              |       |extractive_match|0.838|±  |0.0165|
|custom:math_500:0|      1|extractive_match|0.838|±  |0.0165|
